packages/remotemetrics/events.go

```diff
@@ -56,6 +62,29 @@ type MessageFinalizedMetrics struct {
 	LikeEdgeCount      int       `json:"likeEdgeCount,omitempty" bson:"likeEdgeCount"`
 }
 
+// MessageScheduledMetrics defines the scheduling message confirmation metrics record that is sent to remote logger
+type MessageScheduledMetrics struct {
+	Type               string    `json:"type" bson:"type"`
+	NodeID             string    `json:"nodeID" bson:"nodeID"`
+	MetricsLevel       uint8     `json:"metricsLevel" bson:"metricsLevel"`
+	MessageID          string    `json:"messageID" bson:"messageID"`
+	IssuedTimestamp    time.Time `json:"issuedTimestamp" bson:"issuedTimestamp"`
+	SolidTimestamp     time.Time `json:"solidTimestamp,omitempty" bson:"solidTimestamp"`
+	ScheduledTimestamp time.Time `json:"scheduledTimestamp,omitempty" bson:"scheduledTimestamp"`
+	BookedTimestamp    time.Time `json:"bookedTimestamp" bson:"bookedTimestamp"`
+	QueuedTimestamp    time.Time `json:"queuedTimestamp" bson:"queuedTimestamp"`
```

Comment:
Queued timestamp is the same as booked timestamp, right? I think we should only store one value.

Reply:
As per our DM we should leave this since there might be a difference